docs/apache-airflow/howto/set-config.rst

```diff
@@ -38,6 +38,19 @@ or by creating a corresponding environment variable:
 
     export AIRFLOW__DATABASE__SQL_ALCHEMY_CONN=my_conn_string
 
+Note that when the section name has a dot in it, you must replace it with an underscore when setting the env var.
+For example consider section ``providers.odbc``:
+
+.. code-block:: ini
+
+    [providers.odbc]
+    allow_driver_in_extra = true
+
+.. code-block:: bash
+
+    export AIRFLOW__PROVIDERS_ODBC__ALLOW_DRIVER_IN_EXTRA=true
+
+
```

Comment:
i am not sure if we should have providers settings in Airflow core docs..
this may create issues if for some reason in future the provider will remove this.
This also goes against our goal of eventually extracting the provider code base out of airflow core (at least I think we want to be ready for a day where we might want to do it)

If we must do this I think it's preferred to add a note with a link to the relevant information in the provider docs (external link!) thus treating the providers doc as external resource like any other package.

Reply:
So... this is meant to be an example.  If you like i can make it not refer to specific case.  i thought about this.